pkg/ingress/model_build_listener.go

```diff
@@ -34,9 +38,12 @@ func (t *defaultModelBuildTask) buildListenerSpec(ctx context.Context, lbARN cor
 	if err != nil {
 		return elbv2model.ListenerSpec{}, err
 	}
-	tags, err := t.buildListenerTags(ctx, ingList)
-	if err != nil {
-		return elbv2model.ListenerSpec{}, err
+	tags := map[string]string{}
+	if t.featureGate.Enabled(EnableListenerRulesTagging) {
+		tags, err = t.buildListenerTags(ctx, ingList)
```

Comment:
actually, let's minimize the changes, we don't need to change the model_builder sections, and let it continue build tags as normal.

These tags built here won't be used in model_deployer anyway when EnableListenerRulesTagging is not enabled.

Reply:
Done. Revoked the changes in model builder side. Thanks for pointing out!